src/app/src/test/kotlin/org/imperial/mrc/hint/integration/clients/ADRClientTests.kt

```diff
@@ -15,7 +15,7 @@ class ADRClientTests
     fun `can parse successful response from ADR`()
     {
         val sut = ADRFuelClient(ConfiguredAppProperties(), "fakekey")
-        val response = sut.get("/organization_list_for_user")
+        val response = sut.get("organization_list_for_user")
```

Comment:
turns out the latest version of the ckan API is sensitive to extra slashes, unlike the previous version